src/Spec.ts

```diff
@@ -408,7 +493,12 @@ export default class Spec {
     try {
       data = yaml.safeLoad(block.textContent!);
     } catch (e) {
```

Comment:
Does this error contain position info we can preserve?

Reply:
Technically yes, though it's not particularly good information. Updated.